### PR TITLE
Fix Label Only Notebook

### LINKED ISSUE #1480: Missing module/folders in Label Only Jupyter Notebook
I'm trying to run the jupyter notebook label_only_membership_inference.ipynb, but I'm unable to the first cell and third cell. Seems that the models folder is not included in the current repo. Unsure if this is a problem on my end or the repo.

Thanks in advance,
Anand

First cell:
import torch
from torch import nn
import numpy as np
from models.mnist import Net

Third cell:
state_dict = torch.load("models/mnist_cnn.pt")
model = Net()
model.load_state_dict(state_dict)

### PR DESCRIPTION
**Pull request by chao1995**
# Description

Fixes #1480

## Type of change

Please check all relevant options.

- [x] Improvement (non-breaking)
- [ ] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by beat-buesser** [comment]
Hi @chao1995 Thank you very much! I'll take a closer look for review soon.

**Review by beat-buesser — APPROVED** [review]
Hi @chao1995 Thank you very much for your contribution to ART! The changes look good to me.

Btw, please let me know if you are interested to work on future projects for ART!

**Comment by chao1995** [comment]
Hi @beat-buesser thank you very much for the review. I have used ART in my research and it really helps me a lot. Will definitely continue to contribute to ART whenever there is an opportunity. Thank you very much!
